napari_btrack/_tests/test_dock_widget.py

```diff
@@ -1,6 +1,7 @@
-from napari_btrack import napari_experimental_provide_dock_widget
 import pytest
 
+from napari_btrack import napari_experimental_provide_dock_widget  # noqa: F401
```

Comment:
This isn't being used currently, not sure if intentional. `# noqa: F401` tells `flake8` not to worry